doc/classes/PopupMenu.xml

```diff
@@ -32,6 +44,8 @@
 			<argument index="2" name="global" type="bool" default="false">
 			</argument>
 			<description>
+				Adds a checkbox and assigns the specified [ShortCut] to it. The label of the checkbox will be the [ShortCut] name.
+				NOTE: Checkbox does not have a script attached to it. You will have to control their behavior manually.
```

Comment:
You can remove `NOTE: `, and in general, we don't have all caps words like these in the docstrings.

The created checkbox can't have a custom script attached to it, so I would replace the entire line with something like `To control the checkbox, see [method set_item_checked].`